### ISSUE TITLE
Can't type or paste Chr() in the forum's TinyMCE editor

### ISSUE THREAD

**Issue by jornmo** [e107inc/e107#1878]
Hi!

When I/we try to paste or write a text with Chr() in the forums, it does not work. When pasting noting happens, and when writing it, the post is erased.

**Comment by rica-carv** [comment]
[Chr()](https://www.google.pt/url?sa=t&rct=j&q=&esrc=s&source=web&cd=5&cad=rja&uact=8&ved=0ahUKEwj6-PP7r57PAhWE1hQKHXsIClsQFggtMAQ&url=http%3A%2F%2Fphp.net%2Fmanual%2Fen%2Ffunction.chr.php&usg=AFQjCNF_LpSUouR8dgJCPLV8FAYzUHoA4g) is PHP code, right?
So, i wonder why it's deleted, uhhh???? Maybe for some weird security reason, no?

Why not use the [ASCII or HTML equivalent](http://www.atwebresults.com/ascii-codes.php?type=2)???

**Comment by Moc** [comment]
What exactly are you trying to achieve? Can you copy the text you'd like to write? 
You cannot use PHP functions in any part of user-input areas in e107, due to security reasons.

**Comment by jornmo** [comment]
We're trying to set up a programming community. Being able to paste source code snippets is mandatory.

**Comment by rica-carv** [comment]
Even the chr() php code inside the code tags doesn't work? Using the[ code sample TinyMCE plugin](https://www.tinymce.com/docs/plugins/codesample/)?

**Comment by Moc** [comment]
@CaMer0n How would one approach posting code extracts into forums for a programming community for example?

**Comment by rica-carv** [comment]
Indeed, the TinyMCE page is a little sparse on how to setup plugins inside TinyMCE.....
